### PR TITLE
Set components

### LINKED ISSUE #21: Implement Redux cycle for storing information queried from BE
Reducers: Source language, Target Language, Methods, Results
Actions: TBD

### LINKED ISSUE #24: Successfully query something from the BE in the FE and display it's contents

### PR DESCRIPTION
**Pull request by VeeAndrade**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [ ] Link to project
- [ ] Added to Readme
​
### Issues Resolved
Resolves #21 Resolves #24 
​
### Problem Addressed
​Unable to successfully display queried data
### Solution Implemented
​Restructured query function
### Screenshots (optional)
Screen Shot 2020-04-12 at 12 17 12 PM
### Other Notes

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import TranslationBlock from '../TranslationBlock/TranslationBlock';
+import ResultsContainer from '../../containers/ResultsContainer';
+import ApolloClient from 'apollo-boost';
+import { ApolloProvider } from '@apollo/react-hooks';
+import '../../styles/App.scss';
+
+const client = new ApolloClient({
+  uri: 'https://rosetta-server.herokuapp.com/graphql',
+});
+
```
Comment (mintona) [inline src/components/App/App.jsx:12]: Looking forward to digging into this shortly!

File: src/components/IntroCard/IntroCard.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../../styles/IntroCard.scss';
+
+const IntroCard = () => {
```
Comment (mintona) [inline src/components/IntroCard/IntroCard.jsx:4]: The start of these cards is looking good.

File: src/components/TranslationBlock/TranslationBlock.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import '../../styles/TranslationBlock.scss';
+import SourceBlock from '../SourceBlock/SourceBlock';
+import { getAllLanguages } from '../../queries'
+import { useQuery } from "@apollo/react-hooks";
+import { connect } from 'react-redux'
+import { setSourceLanguage, setSourceMethod, setTargetLanguage } from '../../actions'
+
+const TranslationBlock = (props) => {
+  let { data, loading, error } = useQuery(getAllLanguages)
```
Comment (mintona) [inline src/components/TranslationBlock/TranslationBlock.jsx:10]: Yes, the query!

**Review by mintona — APPROVED** [review]
Nice job, Vee. Looking forward to working with you more on this soon!

**Review by jobannon — APPROVED** [review]
Interesting to view and see the integration of GraphQL.  Looking forward to viewing how components interact and building a few.

**Pull request merged**